spark-connector/spark-connector/src/main/java/com/datastrato/gravitino/spark/connector/iceberg/GravitinoIcebergCatalog.java

```diff
@@ -96,6 +103,28 @@ public UnboundFunction loadFunction(Identifier ident) throws NoSuchFunctionExcep
     return ((SparkCatalog) sparkCatalog).loadFunction(ident);
   }
 
+  @SuppressWarnings("deprecation")
+  @Override
+  public StagedTable stageCreate(
```

Comment:
Gravitino doesn't support stage create for now, this will bypass the gravitino when `create&replace table as select`, I prefer to delay support of stage create for now. WDYT? cc @jerryshao

Reply:
> Gravitino doesn't support stage create for now, this will bypass the gravitino when `create&replace table as select`, I prefer to delay support of stage create for now. WDYT? cc @jerryshao

But rest apis don't support CTAS or RTAS either. Is the purpose an audit?